app/celery/tasks.py

```diff
@@ -697,19 +697,19 @@ def choose_sending_queue(process_type: str, notif_type: str, notifications_count
     queue: Optional[str] = process_type
 
     if notifications_count >= large_csv_threshold:
-        queue = QueueNames.BULK
+        queue = QueueNames.SEND_SMS_LOW if notif_type == SMS_TYPE else QueueNames.BULK
     # If priority is slow/bulk, but lower than threshold, let's make it
     # faster by switching to normal queue.
     elif process_type == BULK:
-        queue = QueueNames.SEND_NORMAL_QUEUE.format(notif_type)
+        queue = QueueNames.SEND_SMS_MEDIUM if notif_type == SMS_TYPE else QueueNames.SEND_NORMAL_QUEUE.format(notif_type)
     else:
         # If the size isn't a concern, fall back to the template's process type.
         if process_type == PRIORITY:
-            queue = QueueNames.PRIORITY
+            queue = QueueNames.SEND_SMS_HIGH if notif_type == SMS_TYPE else QueueNames.PRIORITY
         elif process_type == BULK:
-            queue = QueueNames.BULK
+            queue = QueueNames.SEND_SMS_LOW if notif_type == SMS_TYPE else QueueNames.BULK
         else:
-            queue = QueueNames.SEND_NORMAL_QUEUE.format(notif_type)
+            queue = QueueNames.SEND_SMS_MEDIUM if notif_type == SMS_TYPE else QueueNames.SEND_NORMAL_QUEUE.format(notif_type)
     return queue
```

Comment:
This is getting more complex and would be worth having its own data structure (like a dictionary) or class to express such logic.

Reply:
yeah... I spent far too much time refactoring a bit and adding a `Priorities` class and a dict `DELIVERY_QUEUES` to the `QueueNames` class. I think this makes it easier to understand what queues are going to be used.